conf/cassandra.yaml

```diff
@@ -1455,15 +1455,25 @@ enable_drop_compact_storage: false
 #    - 127.0.0.0/31
 
 # Enables tracking warnings/aborts across all replicas for reporting back to client.
-# Scheduled to enable in 4.2
 # See: CASSANDRA-16850
-# See: tombstone_warn_threshold, tombstone_failure_threshold, client_large_read_warn_threshold_kb, and client_large_read_abort_threshold_kb
-#client_track_warnings_enabled: false
-
-# When client_track_warnings_enabled: true, this tracks the materialized size of a query on the
-# coordinator. If client_large_read_warn_threshold_kb is greater than 0, this will emit a warning
-# to clients with details on what query triggered this as well as the size of the result set; if
-# client_large_read_abort_threshold_kb is greater than 0, this will abort the query after it
-# has exceeded this threshold, returning a read error to the user.
-#client_large_read_warn_threshold_kb: 0
-#client_large_read_abort_threshold_kb: 0
+#track_warnings:
```

Comment:
nit: What about something like `heap_warnings`/`read_size_warnings`/`query_size_warnings`? Not super sold on the token `track`, given it's a verb :/

Reply:
those are a bit too specific; as this is a feature to enable replicas to notify users about behavior; so any query (read/write/ddl) could be in scope; the best I can think of is "query_warnings"?

Reply:
for example, what blocks us from using this in alter to warn/abort the mutation if a local SSTable matches a condition (looking at drop compact storage)?

Reply:
`query_warnings` WFM

Reply:
...and the related interface from `TrackWarnings` -> `QueryWarnings`, etc.

Reply:
@bdeggleston thoughts?